test/basic.html

```diff
@@ -58,7 +55,7 @@
       var f, img;
       setup(function () {
         f = fixture('basic');
-        img = f.querySelector('iron-image');
+        img = f.$$('iron-image');
```

Comment:
yeah! https://www.polymer-project.org/1.0/docs/devguide/local-dom.html#work-with-local-dom (the last thing in that section).

Is it weird? `$$` goes through the shadow boundary, which this test needs